server/atelier-queries.js

```diff
@@ -47,13 +47,37 @@ const getProductReviewsMeta = (productId, headers) => {
 
 // Returns the questions as an array
 const getProductQuestions = (productId, headers) => {
-  return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${productId}`, headers)
+  return axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=${productId}&count=5000`, headers)
```

Comment:
count=5000 as a way to get ALL the questions, one presumes?
Seems good as a hack to hit BMR.
In a real page, this could come back to bite us a couple ways: A major product could get more than 5,000 questions. And since we're being timed, doing this in such big bites might ping out time if we did get a product with a lot of questions. But for BMR, this is good.

Reply:
I took a quick look over at Amazon to see if super-popular products ever got... how many questions, actually?
And actually, at a quick skim, nothing got close to 5,000. Not duct tape. Not pokemon. Not a super-popular chair that I clicked on since it had 20,000 reviews and... not a single question, somehow?

Reply:
Good point, I will change this to something lower!